python-spec/src/somacore/data.py

```diff
@@ -52,10 +52,10 @@ def create(
         [lifecycle: experimental]
 
         The schema of the created dataframe will include a column named
-        ``soma_joinid`` of type ``pyarrow.int64``. If a ``soma_joinid`` column
-        is present in the provided schema, it must be of the correct type. If no
-        ``soma_joinid`` column is provided, one will be added. This column may
-        be used as an indexed column.
+        ``soma_joinid`` of type ``pyarrow.int64``, with negative values disallowed.
+        If a ``soma_joinid`` column is present in the provided schema, it must be of
+        the correct type.  If no ``soma_joinid`` column is provided, one will be
+        added. This column may be used as an indexed column.
```

Comment:
just a reflow with a changed word or two:

```suggestion
        ``soma_joinid`` of type ``pyarrow.int64``, with negative values
        disallowed.  If a ``soma_joinid`` column is present in the provided
        schema, it must be of the correct type.  If no ``soma_joinid`` column
        is provided, one will be added.  It may be used as an indexed column.
```